Source/WebCore/loader/CookieJar.cpp

```diff
@@ -113,7 +113,7 @@ CookieRequestHeaderFieldProxy CookieJar::cookieRequestHeaderFieldProxy(const Doc
 
     auto pageID = document.pageID();
     std::optional<FrameIdentifier> frameID;
-    if (auto* frame = document.frame())
+    if (RefPtr frame = document.frame())
```

Comment:
False positive?

Reply:
Yeah, I also think this and the others are false positives. I did ask in our channel about this earlier. I'll remove these changes from this patch.